dao/src/test/resources/logback.xml

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
perhaps name this file `logback-test.xml`, to help poor souls that run from unpopular IDE